packages/ledger-icrc/src/utils/ledger.utils.ts

```diff
@@ -28,7 +28,9 @@ export const encodeIcrcAccount = ({
 
   const removeLeadingZeros = (text: string): string => text.replace(/^0+/, "");
 
-  const subaccountText = removeLeadingZeros(uint8ArrayToHexString(subaccount));
+  const subaccountText = removeLeadingZeros(
+    uint8ArrayToHexString(Uint8Array.from(subaccount)),
```

Comment:
subaccount ca be undefined? if yes, we should add a check no?

Reply:
Yes, thanks! I wonder why TS didn't catch this...

Reply:
I found why TS didn't catch it.

There is a check on `subaccount` before:

```
  if (isNullish(subaccount)) {
    return owner.toText();
  }
```

That means that subaccount is already defined in that line.